src/sentry/static/sentry/less/base.less

```diff
@@ -142,6 +142,7 @@ figure {
 
 img {
   vertical-align: middle;
+  max-width: 100%;
```

Comment:
This was in the `misc.less` file but that all needs to be moved into other files. 

In this case, html element styles belong in `base.less`